account_cutoff_accrual_base/models/account_cutoff.py

```diff
@@ -12,7 +12,7 @@ class AccountCutOff(models.Model):
     @api.model
     def _default_cutoff_account_id(self):
         account_id = super(AccountCutOff, self)._default_cutoff_account_id()
-        cutoff_type = self.env.context.get('default_cutoff_type')
+        cutoff_type = self.env.context.get('cutoff_type')
```

Comment:
Sure is this the problem? I find other functions with this logic, Travis is red too

Reply:
`default_cutoff_type` is never set in context.

I will check travis.

Thanks

Reply:
Fixed

Reply:
:+1: